tests/openkit-shared-tests/Core/Configuration/OpenKitConfigurationTest.cs

```diff
@@ -172,6 +175,25 @@ public void ADefaultConstructedConfigurationUsesStrictTrustManager()
             Assert.That(sslTrustManager, Is.InstanceOf<SSLStrictTrustManager>());
         }
 
+        [Test]
+        public void GetApplicationID()
+        {
+            // given
+            var target = CreateDefaultConfig();
+
+            // then
+            Assert.That(target.ApplicationID, Is.EqualTo("/App_ID%"));
+        }
+
+        [Test]
+        public void GetApplicationIDPercentEncodedDoesPropperEncoding()

```

Comment:
typo:
GetApplicationIDPercentEncodedDoesProperEncoding

Reply:
True, unfortunately, this was also in Java -> for Java, this has been fixed already.

Reply:
Fixed in .NET also